MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1553,18 +1555,14 @@ ydb_fork_n_core()
 
 	void ydb_fork_n_core(void)
 
-The core of a process can help debug application code, for example to
-troubleshoot an out-of-design condition. However, generating a core
-terminates the process, and furthermore, if the process is last
-process with a database file open, that file can be left in an
-“unclean” state, even if a state that is easily recovered; and
-cleaning up open database files before generating the core perturbs
-the process state to be debugged.
-
-When a process executes :code:`ydb_fork_n_core()`, the process forks. The
-child process does essential cleanup (such as erasing encryption
-passphrases) and sends itself a signal to generate a core and
-terminate, while the parent process continues execution.
+A core is a snapshot of a process, to help debug application code, for
+example to troubleshoot an out-of-design condition.  When a process
+executes :code:`ydb_fork_n_core()`, it forks. The child process does
+essential cleanup (such as erasing encryption passphrases) and sends
+itself a signal to generate a core and terminate. On termination of
+the child process, the parent process continues execution. Note that
+depending on the nature of the condition necessitating a core, an exit
```

Comment:
exit() call

Note an exit call will drive the YottaDB exit handler which attempts a clean shutdown of any open databases or devices.